SharpXMPP.Shared/SharpXMPP.Shared.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <VersionPrefix>0.0.3</VersionPrefix>
-    <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
+    <TargetFrameworks>netstandard1.6;netstandard2.0;netstandard2.1;net451;net472;net48</TargetFrameworks>
```

Comment:
Sorry, what's the point of building both for `netstandard1.6` and any more recent ones? I'd shorten this list to `netstandard1.6;net451`, and can't see a point of separate builds for others. It will just bloat our package, no?